### DISCUSSION TITLE
Access defined globals inside a story

### DISCUSSION THREAD

**Discussion by seppzero** [storybookjs/storybook#14072] [Help] [answered]
Is there a way to access globals defined for the toolbar inside a Story?

I saw there is a hook **useGlobals** but it seems only working for addons.
In case of using it inside a Story, im getting the following TypeError:

TypeError: Cannot read property 'updateGlobals' of undefined

**Comment by shilman** [comment, marked as answer]
If you just want access to the current values, the second argument to a function is a StoryContext which contains a `globals` field

```js
export const MyStory = (args, { globals }) => {  .... }
```

**Reply by seppzero** [reply]
Perfect. Thanks :)
